keps/1282-pods-ready-requeue-strategy/README.md

```diff
@@ -75,25 +105,69 @@ Consider including folks who also work outside the SIG or subproject.
 
 ### API Changes
 
-Add an additional field to the Kueue ConfigMap to allow administrators to specify what timestamp to consider during queue sorting (under the pre-existing waitForPodsReady block).
+#### KueueConfig
+
+Add an additional field to the KueueConfig to allow administrators to specify what timestamp to consider during queue sorting (under the pre-existing waitForPodsReady block).
 
 Possible settings:
 
 * `Eviction` (Back of queue)
 * `Creation` (Front of queue)
 
-```yaml
-kind: ConfigMap
-metadata:
-  name: kueue-manager-config
-  namespace: kueue-system
-data:
-  controller_manager_config.yaml: |
-    apiVersion: config.kueue.x-k8s.io/v1beta1
-    kind: Configuration
-    # ...
-    waitForPodsReady:
-      requeuingTimestamp: Creation | Eviction # <-- New field
+```go
+type WaitForPodsReady struct {
+	...
+	// requeuingStrategy defines the strategy for requeuing a Workload
+	// +optional
+	RequeuingStrategy *RequeuingStrategy `json:"requeuingStrategy,omitempty"`
+}
+
+type RequeuingStrategy struct {
+	// timestamp defines the timestamp used for requeuing a Workload
+	// that was evicted due to Pod readiness. Defaults to Eviction.
+	// +optional
+	Timestamp *RequeuingTimestamp `json:"timestamp,omitempty"`
+	
+	// backOffLimitCount defines the maximum number of requeuing retries.
+	// When the number is reached, the workload is deactivated.	
```

Comment:
What do you mean by deactivated?

Reply:
I  meant using "Suspend a running Job without requeueing" feature.

Reply:
```suggestion
	// When the number is reached, the workload is deactivated (`.spec.active=false`).	
```

Reply:
Done.